src/wallet/wallet.cpp

```diff
@@ -1709,8 +1709,8 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
 {
     using Clock = std::chrono::steady_clock;
```

Comment:
In commit "wallet test: Add unit test for wallet scan save_progress option" (18926f9259ed8656645602807a944ca49a4e2138)

This declaration is unused and looks like it needs to be deleted to avoid a fatal warning:

```
wallet/wallet.cpp:1710:11: error: unused type alias 'Clock' [-Werror,-Wunused-local-typedef]
    using Clock = std::chrono::steady_clock;
```

https://cirrus-ci.com/task/4683407052505088?logs=ci#L1338

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25036/commits/86698868cdb878140c7d6a3909d6aa6f024d8cbf. Thanks